setup/install/providers/kubernetes-v2/aws-eks.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: single
+title:  "Amazon EKS"
+sidebar:
+  nav: setup
+---
+
+
+
+{% include toc %}
+
+
+
+
+
+> Before you proceed with installing Spinnaker on EKS, we strongly recommend that you familiarize yourself with [Amazon EKS concepts](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
+
+
+
+Below are the instructions to be followed if you want to configure [Kubernetes V2 (manifest based) Clouddriver](/setup/install/providers/kubernetes-v2) 
+Spinnaker to deploy to [Amazon EKS](https://aws.amazon.com/eks/).
```

Comment:
You can delete this whole paragraph. If you add the verb to the title, this becomes redundant. 

Also, as I think @lwander  indicated, this should be presented as adding a *provider* that happens to be EKS, and deploying Spinnaker to it is incidental to that.